viewing_party/party.py

```diff
@@ -0,0 +1,168 @@
+# Wave 1 #
+def create_movie(title, genre, rating):
+    if title == None:
+        return None
+    if genre == None:
+        return None
+    if rating == None:
+        return None
```

Comment:
Here's one way you might consider refactoring that's very pythonic
```suggestion
    if not title or not genre or not rating:
        return None
```